extension.neon

```diff
@@ -1,3 +1,6 @@
+parameters:
+    bootstrap: %rootDir%/../../saschaegerer/phpstan-typo3/src/PhpstanTypo3Bootstrap.php
```

Comment:
You should continue to make this optional, otherwise this package cannot be used with `nimut/testing-framework` which does not provide the `TYPO3\TestingFramework\Core\Testbase` class.

Reply:
You can still overwrite this with your own bootstrap. So if you define a bootstrap in your config it should not load this one, right?

Reply:
Correct but I don't need any bootstrap.

Reply:
Is the updated pull request fine for you @mbrodala

Reply:
Looks good, I'll do a test run.